### PR TITLE
Okta-475629 Updates as per new Rate Limits Monitoring widget

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updated Rate Limits content to remove references to notification banner for warnings and violations, as well as add content. on the new Rate Limit Monitoring widget.
- **Is this PR related to a Monolith release?** ?

[https://621fba76d6d6495d73b773fe--reverent-murdock-829d24.netlify.app/docs/reference/rate-limits/#rate-limit-monitoring-widget](https://621fba76d6d6495d73b773fe--reverent-murdock-829d24.netlify.app/docs/reference/rate-limits/#rate-limit-monitoring-widget)

### Resolves:

* [OKTA-475629](https://oktainc.atlassian.net/browse/OKTA-475629)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ If any org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 > * You can expand the Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
+## Rate Limit Monitoring widget
+
+The Admin Console tracks any rate-limit warnings or violations directly in a Rate Limit Monitoring widget available on the Admin Dashboard. By default, only the last hour of warnings or violations appear, but you can also check for events within the last 24 hours or the last 7 days from the drop-down menu. Selecting **View** at the top of the widget takes you to the [Rate Limits dashboard](/docs/reference/rl-dashboard/) for further investigation. If individual rate-limit violations appear in the widget, you can access affected API usage in the Rate Limits Dashboard by clicking the API link in the widget.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:37]: InfoDev ?recently? changed references of drop-down to dropdown.....
```suggestion
The Admin Console tracks any rate-limit warnings or violations directly in a Rate Limit Monitoring widget available on the Admin Dashboard. By default, only the last hour of warnings or violations appear, but you can also check for events within the last 24 hours or the last 7 days from the dropdown menu. Selecting **View** at the top of the widget takes you to the [Rate Limits dashboard](/docs/reference/rl-dashboard/) for further investigation. If individual rate-limit violations appear in the widget, you can access affected API usage in the Rate Limits Dashboard by clicking the API link in the widget.
```

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Currently, the client-based rate limits apply to the `/login/login.htm` endpoint
 
 Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket, for example, `/oauth2/${authorizationServerId}/v1` (Custom Authorization Server) or `/oauth2/v1` (Okta Org Authorization Server). The per minute rate limits on these endpoints apply across an Okta tenant.
 
-Curently, you cannot update the per client rate limit. Every client ID/IP/device identifier combination (for the OAuth 2.0 `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of five concurrent requests.
+Currently, you cannot update the per client rate limit. Every client ID/IP/device identifier combination (for the OAuth 2.0 `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of five concurrent requests.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:24]: suggested contraction for conversational tone.

```suggestion
Currently, you can't update the per client rate limit. Every client ID/IP/device identifier combination (for the OAuth 2.0 `/authorize` endpoint) or IP/device identifier combination (for the `/login/login.htm` endpoint) gets 60 total requests per minute and a maximum of five concurrent requests.
```

**Review by susanharper-okta — APPROVED** [review]
two really minor nits. approving to not hold you up.

File: packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md [hunk 3]
```diff
@@ -93,18 +93,17 @@ The client-based rate limit framework can operate in one of three modes:
...
+If an org's client-based limit was previously set to **Enforce limit and log per client (recommended)** mode, the setting remains as is. For this mode setting, admins need to monitor System Log events for [`system.client.rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level) and [`system.client.concurrency_rate_limit.notification`](/docs/reference/rl-system-log-events/#web-request-rate-limits-client-level). If admins don't see any of those events in the System Log, then no action is needed. If admins see these notifications sporadically from just a small number of different users, then those users may be doing something scripted or automated. In such cases, admins can choose to either investigate the issue or take no action. However, if admins see widespread notifications from a large number of users, then it is likely that the application may have an issue, in which case admins need to troubleshoot and make changes to the application where needed.
```
Comment (peteryaworsky-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md:106]: When the setting is in "Enforce limit and log per client" mode the relevant events are `system.client.rate_limit.violation` and `system.client.concurrency_rate_limit.violation`

**Pull request merged**